tests/globalhelper/runhelper.go

```diff
@@ -47,5 +48,44 @@ func LaunchTests(testSuites []string, skipRegEx string) error {
 	cmd.Args = append(cmd.Args, testArgs...)
 	cmd.Dir = Configuration.General.TnfRepoPath
 
+	if os.Getenv("DEBUG_TNF") == "true" {
+		os.Setenv("LOG_LEVEL", "trace")
+
+		buildNumber, present := os.LookupEnv("BUILD_NUMBER")
+
+		var debugFolderName string
+
+		if present {
+			debugFolderName = "Debug-" + buildNumber
```

Comment:
No need for build number. Let's just overwrite debugs after every run.

Reply:
I think we should save those logs for a certain amount of time, so we would be able to
check those logs.
if we will overwrite them, then we lost the logs of the build.

or we are saving them in a different place?

Reply:
done, we need to save the Debug folder to Jenkins artifacts.